styles/ollama/catppuccin.user.css

```
@@ -0,0 +1,198 @@
+    @icon-filter: @catppuccin[@@lookup][@icon_filter];
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+    & when not (@lookup = latte) {
+      img[src="/public/ollama.png"], img[src="https://ollama.com/public/assets/c889cc0d-cb83-4c46-a98e-0d0e273151b9/42f6b28d-9117-48cd-ac0d-44baaf5c178e.png"] {
+filter: @icon-filter;
+// background-color: @red;
+      }
+    }
+    .text-black, .text-black  * {
+      color: @text ;
+    }
+    .text-white {
+      color: @text;
+    }
+    .text-gray-900 {
+      color: @subtext1;
+    }
+
+    .bg-black {
+      background-color: @surface0;
+    }
+    .bg-neutral-50 {
+      background-color: @surface0;
+    }
+    .bg-green-50 {
+      background-color: @green;
+    }
+    .text-green-700 {
+      color: darken(@green, 30%)
+    }
+    li::marker {
+      color: @text !important;
+    }
+    .divide-gray-200 > * + * {
+border-color: @accent-color !important;
+    }
+    .border {
+      border-color: @accent-color;
+    }
+    .divide-neutral-200 > * + * {
+      border-color: @accent-color !important;
+          }
+    .border-neutral-200 {
+      border-color: @accent-color;
+    }
+    ul {
+      color: @text !important;
+    }
+    #readme > *, #display > *, #file-explorer > *, #readme, #display  {
+      color: @text !important;
+    }
+     .bg-neutral-100 pre, .bg-neutral-100 .command {
+      background-color: unset !important;
+     }
+    code, pre {
+      background-color: @mantle;
+      color: @text !important;  
+    }
+    .bg-yellow-50 {
+      background-color: @yellow;
+    }
+    .text-yellow-900 {
+      color: darken(@yellow, 30%);
+    }
+    button {
+      // background-color: @surface2;
+      color: @text;
+    }
+    input, select ,textarea {
+      background-color: @mantle;
+      color: @text;
+    }
+    .text-neutral-500 {
+      color: @subtext0
+    }
+    .text-neutral-900, .text-neutral-800 {
+      color: @subtext1 !important;
+    }
+    .text-neutral-400 {
+      color: @subtext0
+    }
+    .bg-neutral-800, .bg-neutral-700 {
+      background-color: @surface1;
+    }
+    .bg-neutral-100  {
+      background-color: @surface2 !important;
+    }
+    .hover\:bg-neutral-100:hover {
+      background-color: @surface2 !important;
+    }
+    .command {
+      color: @text;
+    }
+    .ring-neutral-300 {
+      --tw-ring-color: @accent-color !important
+    }
+    .ring-blue-300 {
+      --tw-ring-color: @accent-color !important
+    }
+    .truncate {
+      color: @text;
+    }
+    .bg-indigo-50 {
+      background-color: @accent-color;
+    }
+    .text-indigo-600, .text-blue-500 {
+      color: darken(@accent-color,20%)
+    }
+    .text-blue-600 {
+      color: darken(@accent-color, 25%);
+      background-color: @accent-color;
+    }
+    background-color: @crust;
```

```suggestion
    background-color: @base;
```
The main background color for any website should always be base, then mantle (as you have it below with .bg-white) should be for headers and footers.